### PR TITLE
Fix null error on the content field of project resource

### PR DESCRIPTION
**Pull request by ke4**
Changes:
- Just a small fix for the null error when trying to edit a suggested project

### CODE REVIEW

File: client/src/app/project-form/project-form.component.ts [hunk 1]
```diff
@@ -155,7 +155,7 @@ export class ProjectFormComponent implements OnInit {
           console.log('Retrieved project', projectResource);
           this.projectResource = projectResource;
           if (projectResource && projectResource.content &&
-            !projectResource.content.hasOwnProperty('describedBy') || !projectResource.content.hasOwnProperty('schema_type')) {
+            (!projectResource.content.hasOwnProperty('describedBy') || !projectResource.content.hasOwnProperty('schema_type'))) {
```
Comment (jacobwindsor) [inline client/src/app/project-form/project-form.component.ts:158]: This is fine but I think you can skip the first bit and do `projectResource?.content?.hasOwnProperty('describedBy')`

**Review by jacobwindsor — CHANGES_REQUESTED** [review]

File: client/src/app/project-form/project-form.component.ts (hunk 1 above)
Comment (ke4) [reply client/src/app/project-form/project-form.component.ts:158]: Yes, thanks for that. I forgot the angular way. ;-)
I am going to refactor it like that.
We are also trying to add a test with @amnonkhen.
It is not straightforward to me, but hopefully we can do it.

Comment (jacobwindsor) [reply client/src/app/project-form/project-form.component.ts:158]: It's not anything angular, it's just new JS notation. Sounds like a good plan!

Comment (amnonkhen) [reply client/src/app/project-form/project-form.component.ts:158]: Both are good practice.
Elvis ( ?. ) is always good, and adding unit tests is always good.
I feel we should have a workshop about Angular unit testing.

**Review by jacobwindsor — APPROVED** [review]

**Review by yusra-haider — APPROVED** [review]

**Pull request merged**